doc/adrs/125_add_the_ability_to_reuse_both_the_sirius_web_frontend_and_backend_easily.adoc

```diff
@@ -0,0 +1,37 @@
+= ADR-125 - Add the ability to reuse both the Sirius Web frontend and backend easily
+
+== Context
+
+To get started with an application built on top of Sirius Web, some complex steps needs to be performed.
+On the frontend, things have been simplified with the introduction of `@eclipse-sirius/sirius-web-application`.
+On the backend, consumers still need to depend on every single one of our components by themselves.
+
+While consumers want to be able to reuse Sirius Web off the shelf very easily, they also want to control what can appear in the end result.
+
+== Decision
+
+We will introduce a dedicated starter for the Sirius Web backend named `sirius-web-application-starter`.
+This starter will depend on everything required to get started with both Sirius Web and Sirius Components.
+Applications depending on this starter will retrieve every feature from Sirius Components and Sirius Web at once.
```

Comment:
When you said every feature, do you mean only the strict minimum ?

Reply:
I will make that a bit more explicit, everything will be included in the classpath but not everything will be activated by default.